src/Files.App/Filesystem/StorageEnumerators/UniversalStorageEnumerator.cs

```diff
@@ -1,36 +1,21 @@
 // Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
-using CommunityToolkit.Mvvm.DependencyInjection;
-using Files.App.Extensions;
 using Files.App.Filesystem.StorageItems;
-using Files.App.Helpers;
 using Files.Backend.Helpers;
-using Files.Backend.Services.Settings;
 using Microsoft.UI.Xaml.Media.Imaging;
-using System;
-using System.Collections.Generic;
 using System.IO;
-using System.Linq;
-using System.Threading;
-using System.Threading.Tasks;
 using Windows.Storage;
 
 namespace Files.App.Filesystem.StorageEnumerators
 {
 	public static class UniversalStorageEnumerator
 	{
-		public static async Task<List<ListedItem>> ListEntries(
-			BaseStorageFolder rootFolder,
-			StorageFolderWithPath currentStorageFolder,
-			CancellationToken cancellationToken,
-			int countLimit,
-			Func<List<ListedItem>, Task> intermediateAction,
-			Dictionary<string, BitmapImage> defaultIconPairs = null
-		)
+		public static async Task<List<ListedItem>> ListEntries(BaseStorageFolder rootFolder, StorageFolderWithPath currentStorageFolder, CancellationToken cancellationToken, int countLimit, Func<List<ListedItem>, Task> intermediateAction, Dictionary<string, BitmapImage> defaultIconPairs = null)
```

Comment:
Should this be split onto multiple lines?

Reply:
I don't think so. We hardly add or change existing method's arguments. We can see xml comments instead of lines of arguments.

By the way, I will add XML comments in stage 4.

Reply:
I don't think so.

- We hardly add/change arguments.
- Visual Studio method collapse doesn't work property.
- We can add XML comments instead of just seeing arguments.

By the way, I will add XML comments in stage 4.